.github/workflows/ci.yml

```diff
@@ -56,3 +59,103 @@ jobs:
             -
                 name: "Run friendsofphp/php-cs-fixer"
                 run: "vendor/bin/php-cs-fixer fix --dry-run --diff"
+
+    test:
+        name: "PHP ${{ matrix.php-version }} + Composer v${{ matrix.composer-version }} + MAKER_TEST_VERSION: ${{ matrix.maker-test-version }}"
+
+        runs-on: ubuntu-latest
+
+        services:
+            mysql:
+                image: mysql:5.7
+                env:
+                    MYSQL_ROOT_PASSWORD: root
+                    MYSQL_DATABASE: test_maker
+                ports:
+                    - 3306
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        env:
+            MAKER_TEST_VERSION: ${{ matrix.maker-test-version }}
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.1.33'
+                    - '7.2.5'
+                    - '7.3'
+                    - '7.4'
+                maker-test-version:
+                    - 'stable'
+                allow-failures: [false]
+                composer-version:
+                    - '1'
+                    - '2'
```

Comment:
This really "blows up" the matrix to a lot of cases. I'm trying to think of it's worth it... it seems like it IS a good thing to have tested once, but it seems pretty unlikely that something we do in the future would break on composer 1. I'd vote for just using Composer 2. WDYT?

Reply:
Fine 👍